src/tests/ftest/security/pool_connect_init.yaml

```diff
@@ -4,6 +4,16 @@ hosts:
 timeout: 200
 server_config:
    name: daos_server
+# Uncomment following to force the use of certificates
+# regardless of the launch.py --insecure setting.
+#   transport_config:
+#     allow_insecure: False
+#agent_config:
+#   transport_config:
+#     allow_insecure: False
+#dmg:
+#   transport_config:
+#     allow_insecure: False
```

Comment:
So what's happening in this test? This has neither allow_insecure as True or false. What is the default? Other yaml files above, either have it mentioned as True of false. SO just trying to understand what is happening when it is not mentioned.

Reply:
Certificates will be enabled by default including this test, the commented-out lines providing a reference in case there is a need to run with insecure-mode.